docs/UserGuide.md

```diff
@@ -482,6 +476,18 @@ For an interview,
   "status" : "PENDING"
 }]
 ```
+*Note that interview does not save a candidate but its unique ID generated within the application.*
+
+Certain fields are editable directly without repercussions <U>as long as the format is valid (as shown above)</U>, like **date**, **startTime**, **duration** and **status** in `Interviews.json`
+However, the same cannot be said for fields of different files sharing the same information, like **positions** in `Candidates.json` and the entire `Positions.json` file.
+Any discrepancy could cause HR Manager to display misrepresented information.
+
+<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
```

Comment:
![image](https://user-images.githubusercontent.com/61644924/140009848-5365a4f4-ceeb-4c4d-b4d6-8a3e75ce8934.png)

Same thing here. Also should add a newline after the caution

Reply:
Thanks for catching it! Will change in the next commit

Reply:
and Caution is not bolded